pkg/flow/grpc-logs.go

```diff
@@ -20,7 +24,40 @@ var logsOrderings = []*orderingInfo{
 	},
 }
 
-var logsFilters = map[*filteringInfo]func(query *ent.LogMsgQuery, v string) (*ent.LogMsgQuery, error){}
+var logsFilters = map[*filteringInfo]func(query *ent.LogMsgQuery, v string) (*ent.LogMsgQuery, error){
```

Comment:
This function is full of string constants that we should probably define in a single location.